### DISCUSSION TITLE
How to link to another page/post?

### DISCUSSION THREAD

**Discussion by World-X** [saicaca/fuwari#177] [Q&A] [answered]
Ditto. I want to add a link, whether by the use of Markdown or HTML tags, from a post or page to another page within the same site. Something like this: `[Introduction](/src/posts/introduction.md)` or similar.

**Comment by saicaca** [comment, marked as answer]
The link should be a URL path relative to your root domain, not a file path.

In this case, the generated URL for `/src/posts/introduction.md` will be `https://your-domain.com/posts/introduction/`, so the link here should be `[Introduction](/posts/introduction/)`. If you have set the `base`, make sure to include it in the link as well.

Alternatively, you can try this plugin: https://github.com/vernak2539/astro-rehype-relative-markdown-links

**Reply by World-X** [reply]
Thanks! I experimented a bit, and got it to work. For anyone in the future that may encounter this same question...
To link from `about.md` to `{my_file}.md`, assuming the latter is in `src/content/posts/`, you have to use `../posts/{my_file}/`.
If it's inside a folder, make sure to also include the folder, like `../posts/folder/{my_file}/`.
To link from any post to another post, you have to use `/{base}/posts/{my_file}/` or `/{base}/posts/folder/{my_file}/`, where `{base}` is the one you set in `astro.config.mjs`, just swap `{base}` with whatever you have set over there.

Again, thank you!
